OpenRA.Mods.Common/Lint/CheckDefaultVisibility.cs

```diff
@@ -36,22 +36,22 @@ void Run(Action<string> emitError, Ruleset rules)
 				// Catch TypeDictionary errors.
 				try
 				{
-					var count = actorInfo.Value.TraitInfos<IDefaultVisibilityInfo>().Count();
+					var visibilityTypes = actorInfo.Value.TraitInfos<IDefaultVisibilityInfo>();
 
-					if (count == 0)
+					if (!visibilityTypes.Any())
```

Comment:
Seems like this was de-improved a bit. I suggest pulling out a variable:
`var count = visibilityTypes.Count;`
And then reverting the checks back to as it was before.

Reply:
since I'm calling count already I suppose that makes sense